src/brackets.config.prerelease.json

```diff
@@ -0,0 +1,7 @@
+{
+    "healthDataServerURL"     : "https://health.brackets.io/healthDataLog",
+    "analyticsDataServerURL"  : "https://cc-api-data.adobe.io/ingest",
+    "serviceKey"              : "brackets-service",
+    "environment"             : "production",
```

Comment:
Can it be marked with a different environment `beta` or `experimental` or `prerelease`?

Reply:
I'm not sure what the impact will be in the client code if it is being used. I've just tried to retain it the way it was earlier.
If there are pointers on where all i'll have to look if i change the environment I can take a shot at changing this as well.

Reply:
The only check I know is in HealthDataManager to avoid logging from dev setups. 
```javascript
// Expose a command to test data sending capability, but limit it to dev environment only
    CommandManager.register("Sends health data and Analytics data for testing purpose", "sendHealthData", function() {
        if (brackets.config.environment === "stage") {
            return checkHealthDataSend(true);
        } else {
            return $.Deferred().reject().promise();
        }
    });
```

Reply:
Yes. I saw this as well. And thought in case of pre-release it should go to production. what do you think ?

Reply:
I gave wrong explanation. I had exposed this command to explicitly send analytics data only when we are in dev environment. This block of code has no functionality when in production.

Reply:
yes. i saw that. have made the change for consistency anyways.